virtcontainers/container.go

```diff
@@ -664,6 +664,10 @@ func createContainer(sandbox *Sandbox, contConfig ContainerConfig) (c *Container
 		sandbox.setSandboxPid(c.process.Pid)
 	}
 
+	if ann[annotations.ContainerTypeKey] == string(PodContainer) {
```

Comment:
PodSandbox and PodContainer are both annotations in the kata CLI to get the missing sandbox abstraction from runc compatible command lines. There is no need to use such annotation in virtcontainers, where we know clearly about sandbox vs. containers.

Reply:
which means: once `createContainer()` is called, we're clearly creating a container, I don't need to check `PodContainer` or what, just add the container's pid to cgroup.
Am I clear?~

Reply:
Yes, your understanding is correct.